bin/configs/unmaintained/javascript-apollo.yaml

```diff
@@ -1,5 +1,5 @@
 generatorName: javascript-apollo
-outputDir: samples/client/petstore/javascript-apollo
+outputDir: samples/client/petstore/javascript-apollo-unmaintained
```

Comment:
In the end there was no config clash with [`bin/configs/javascript-apollo.yaml`](https://github.com/OpenAPITools/openapi-generator/blob/8070b4057f3bd43df3e40117906f09d65d36c015/bin/configs/javascript-apollo.yaml), as `unmaintained` configs are not used by `generate-samples.sh`.

Reply:
I'll remove it later.

Reply:
Done via https://github.com/OpenAPITools/openapi-generator/pull/13297